backend/api/Services/MissionRunService.cs

```diff
@@ -219,8 +219,10 @@ private static void SearchByTag(ref IQueryable<MissionRun> missionRuns, string?
                 missionRun =>
                     missionRun.Tasks.Any(
                         task =>
+#pragma warning disable CA1307
                             task.TagId != null
-                            && task.TagId.Contains(tag.Trim(), StringComparison.OrdinalIgnoreCase)
+                            && task.TagId.Contains(tag.Trim())
```

Comment:
```suggestion
                            && task.TagId.ToLower().Contains(tag.ToLower().Trim())
```

Reply:
This is a guid or integer I think so I don't think changing the case would make sense